db/migrate/20170912225408_add_picture_to_photos_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddPictureToPhotosTable < ActiveRecord::Migration[5.0]
```

Comment:
Your app was missing a migration that adds the `picture` field to the `photos` table.